test/evergreen/print_stack_trace.py

```diff
@@ -51,20 +51,25 @@ def _find_debugger(debugger: str):
         """Find the installed debugger."""
         return which(debugger)
 
-    def dump(self, exe_path: str, core_path: str):
+    def dump(self, exe_path: str, core_path: str, dump_all: bool, output_file: str):
         """Dump stack trace."""
         if self.dbg is None:
             sys.exit("Debugger lldb not found,"
                      "skipping dumping of {}".format(core_path))
 
-        cmds = [
-            "thread backtrace all -c 30",
-            "quit"
-        ]
-
+        cmds = []
+        if dump_all:
+            cmds.append("thread apply all backtrace -c 30")
+        else:
+            cmds.append("backtrace -c 30")
+        cmds.append("quit")
+
+        output = None
+        if (output_file):
+            output = open(output_file, "w")
```

Comment:
Should we handle the file open failure case?

Reply:
We should, it should be improved on the `develop` branch first.

Reply:
Updated in 2a71d689365f7ca4041ab1cb7d58f10194f1414e